js/components/atlas.cohort-editor.html

```diff
@@ -10,12 +10,12 @@
 		<!-- ko ifnot:canEdit-->
 		<cohort-expression-viewer params="expression: model.currentCohortDefinition().expression"></cohort-expression-viewer>
 		<!-- /ko -->
-		<div class="modal fade" id="conceptSetSelectorDialog" tabindex="-1" role="dialog">
+		<div class="modal fade" data-bind="modal: showModal" tabindex="-1" role="dialog">
```

Comment:
Thank you so much for ridding the code of element IDs! and using the bindings!! (big heart) (big hug)